frontend/plugins/accounting_ui/src/modules/settings/syncSettings/SyncDealConfigForm.tsx

```
@@ -0,0 +1,383 @@
+              <Form.Item>
+                <Form.Label>{ptype.title}</Form.Label>
+                <Form.Control>
+                  <SelectAccount.FormItem
+                    value={field.value}
+                    onValueChange={field.onChange}
+                    defaultFilter={{ journals: [JournalEnum.BANK, JournalEnum.CASH, JournalEnum.DEBT] }}
+                  />
+                </Form.Control>
+              </Form.Item>
+            )}
+          />
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)